index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta name="description" content="Sample Microverse Hello Project">
+        <meta name="author" content="Albert Mensah-Ansah">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="style.css">
+        <title>My Portfolio</title>
+    </head>
+    <body>
+
+        <nav class="topnav">
+            <div class="welcome">Welcome</div>
+            <div>
+                <i class="fa fa-bars hamburgar"></i>
+            </div>
+        </nav>
+
+        <div class="container">
+            <div class="headline">
+                Hey There. I'm Albert
+                <div class="sub-headline">
+                    I'm a Software Developer
+                </div>
+            </div>
+            
+            <div class="supporting-text">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </div>
```

Comment:
Kindly use the `p` tag instead of the `div` tag because this is a paragraph so it's good to use the paragraph tag, it's more semantic.